python/tskit/trees.py

```diff
@@ -7712,6 +7713,73 @@ def ibd_segments(
             store_pairs=store_pairs,
         )
 
+    def coalescence_time_distribution(
+        self,
+        *,
+        sample_sets=None,
+        weight_func=None,
+        window_breaks=None,
+        blocks_per_window=None,
+    ):
+        """
```

Comment:
As we're not putting this into the public API yet we should remove the docstring. I think you can just insert a blank line with a comment or something to do this?